region/crateria/west.json

```diff
@@ -1389,6 +1389,38 @@
                 }
               ]
             }
+          ],
+          "leaveWithGModeSetup": [
+            {
+              "strats": [
+                {
+                  "name": "Get hit by Waver",
+                  "notable": false,
+                  "requires": [
+                    {"or": [
+                      "h_canDestroyBombWalls",
+                      {"canComeInCharged": {
+                        "fromNode": 1,
+                        "framesRemaining": 180,
+                        "shinesparkFrames": 0
+                      }},
+                      {"and": [
+                        {"comeInWithGMode": {
```

Comment:
Seems extremely niche
I don't know that there is a reason you'd want to use GMode to get GMode on this door.

Reply:
To get into the adjacent room in direct gmode

Reply:
Should probably add a crystal flash to the requirements. As a new strat, really.
Artifical morph->CF->end GMode so the PB breaks the blocks->damage down on waver to appropriate health->leaveWithGModeSetup